packages/sdk/src/streamUtils.ts

```diff
@@ -106,6 +106,7 @@ export function parsedMiniblockToPersistedMiniblock(
 ) {
     if (direction === 'backward') {
         miniblock.header.snapshot = undefined
+        miniblock.header.snapshotHash = undefined
```

Comment:
i think you can delete this line `        miniblock.header.snapshotHash = undefined` - it's fine to know that the header has a snapshot, we were only deleting the snapshot because it's so huge.